runtime/compiler/runtime/RelocationRuntime.cpp

```diff
@@ -1435,3 +1427,32 @@ TR_JITaaSRelocationRuntime::allocateSpaceInDataCache(uintptr_t metaDataSize,
    return newDataStart;
    }
 
+uint8_t *
+TR_JITaaSRelocationRuntime::copyDataToCodeCache(const void *startAddress, size_t totalSize, TR_J9VMBase *fe)
+   {
+   TR::CompilationInfoPerThreadBase *compInfoPT = fe->_compInfoPT;
+   int32_t numReserved;
+   TR::CodeCache *codeCache = NULL;
+   TR::CodeCacheManager *manager = TR::CodeCacheManager::instance();
+   codeCache = manager->reserveCodeCache(false, totalSize, compInfoPT->getCompThreadId(), &numReserved);
```

Comment:
`numReserved` may include reservations done by other threads in system which is possible.
What we want to assert that the current thread does not have anything reserved. I believe that can be tested with `compiler->cg()->getCodeCache()`. If this is NULL, then no code cache is reserved by current compilation.

Reply:
ok, done.

Reply:
One more request: please put the assert before `manager->reserveCodeCache`, otherwise people may get confused as to why there should be no code cache reserved when I just reserved one.

Reply:
done.